go/runtime/bundle/discovery.go

```diff
@@ -28,6 +28,9 @@ const (
 
 	// requestTimeout is the time limit for http client requests.
 	requestTimeout = 10 * time.Second
+
+	// maxBundleSizeBytes is the maximum allowed bundle size in bytes.
+	maxBundleSizeBytes = 20 * 1024 * 1024 // 20 MB
```

Comment:
Can this be made configurable in the configuration file? As long as this is only used for RONLs, it may be fine, but ROFL bundles will be much larger and once this mechanism is also used for ROFLs it will become a problem.

Reply:
I was thinking about that. I changed this to be configurable with default of 20MB.